lib/auth/webauthnwin/ctypes.go

```diff
@@ -35,6 +35,10 @@ type webauthnRPEntityInformation struct {
 	// "Acme Corporation", "Widgets Inc" or "Awesome Site".
 	// This field is required.
 	pwszName *uint16
+
+	// RP icon (previously pwszIcon).
+	// This field is kept just to keep size of struct valid.
+	_ *uint16
```

Comment:
Hmm, if the size of the struct matters, shouldn't we be more explicit? A `*uint16` can have different sizes on different platforms, no?

Reply:
i believe we don't support 32bit tsh at all for windows